MyCryptoHQ/MyCrypto
# ENS Search Styling

Closes #1011. Does a few UX changes to ENS:

## Changes

* Restyled all components to be cleaner and more consistent. See screenshots below.
* Remove the ENS instructions, since you can't actually do them all on v4.
* Reworked some of the copy for consistency. None of it was translated anyway, so I figured it was fair game.
* Reworks components to connect as-needed, rather than having a top-level connected component.
* Changed countdown to better utilize moment, display more human-friendly times.
* Links users to `mycrypto.com/?ensname={name}#ens`. Will follow up with a PR to mycrypto.com that uses the `ensname` param to auto-fill and search for the user.

## Steps to Test

* Go to `/ens` on the `ens-styling` branch
* Search for domains that are in each phase of bidding. You can see available auctions over at https://etherscan.io/ens?filter=auctionskn
* Confirm everything looks good

## Screenshots

### Available

![screen shot 2018-02-07 at 3 18 22 pm](https://user-images.githubusercontent.com/649992/35939626-b2ff72ac-0c1a-11e8-8cd7-6861f8a6fd24.png)


### Auction

![screen shot 2018-02-07 at 3 25 18 pm](https://user-images.githubusercontent.com/649992/35939817-44900dc6-0c1b-11e8-9399-9031fc187548.png)


### Time to Reveal

![screen shot 2018-02-07 at 3 18 04 pm](https://user-images.githubusercontent.com/649992/35939656-c7436278-0c1a-11e8-8a0c-a7ab0af98b5e.png)


### Taken & Info

![screen shot 2018-02-07 at 3 16 42 pm](https://user-images.githubusercontent.com/649992/35939661-cf07ad7a-0c1a-11e8-9a16-eaad0292af68.png)
